bee-node/src/config.rs

```diff
@@ -219,3 +219,31 @@ impl NetworkSpec {
         &self.hrp
     }
 }
+
+#[cfg(test)]
+mod test {
+
+    use super::*;
+
+    #[cfg(feature = "rocksdb")]
+    use bee_storage_rocksdb::storage::Storage;
+    #[cfg(all(feature = "sled", not(feature = "rocksdb")))]
+    use bee_storage_sled::storage::Storage;
+
+    #[test]
+    fn config_files_conformity() -> Result<(), NodeConfigError> {
+        let _ = NodeConfigBuilder::<Storage>::from_file(concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/config.chrysalis-comnet.toml"
+        ))?;
+        let _ = NodeConfigBuilder::<Storage>::from_file(concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/config.chrysalis-devnet.toml"
+        ))?;
+        let _ = NodeConfigBuilder::<Storage>::from_file(concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/config.chrysalis-mainnet.toml"
+        ))?;
+        Ok(())
```

Comment:
```suggestion
        let _ = NodeConfigBuilder::<Storage>::from_file(concat!(
            env!("CARGO_MANIFEST_DIR"),
            "/config.stardust-testnet.toml"
        ))?;
        Ok(())
```

Reply:
Do you mind opening a PR for that right after it's merged ? I don't really want to do a commit in this PR since I'm not going to squash.

Reply:
Sure, I don't mind.